docs_src/utilities/Text.svelte

```diff
@@ -30,3 +30,25 @@
   {`<p class="text-transform--lowercase"></p>
 <p class="text-transform--uppercase"></p>`}
 </Prism>
+
+<h3>Line height</h3>
+<p>Default line-height er baseret på browseren normalt 1.2</p>
+<Card className="bg--graa7 padding-m">
+  <p>Denne brødtekst har en line-height på normal (default). <br> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.</p>
+</Card>
+
+<Prism language="html">
+  {`<div>
+  <p>line-height: normal</p>
+</div>`}
+</Prism>
```

Comment:
ser ingen grund til at have Prism på noget som kun er for at kunne sammeligne.
tænker også en bedre sammenligning vil være at have p tag uden og med `lineheight-m` i samme Card så man kan se forskellen side om side